proceeding/gammapy-icrc2017.tex

```diff
@@ -91,7 +91,7 @@ \section{Introduction}
 \label{sec:intro}
 
 % CTA science tools
-The Cherenkov Telescope Array (CTA) will observe the sky in very-high-energy
+The Cherenkov Telescope Array (CTA) will observe the sky in very high energy (VHE, E > 100$\,$GeV)
```

Comment:
I think the CTA statement is that they will observe at much lower energy, I don't think we can put 100 GeV here.